test/test_serializers.py

```diff
@@ -0,0 +1,62 @@
+"""
+
+    test_parse.py
+
+    Tests for Greynir module
+
+    Copyright(C) 2019 by Miðeind ehf.
+    Original author: Vilhjálmur Þorsteinsson
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""
+
+import json
+
+import pytest
+
+
+@pytest.fixture(scope="module")
+def r():
+    """ Provide a module-scoped Greynir instance as a test fixture """
+    from reynir import Greynir
+    r = Greynir()
+    yield r
+    # Do teardown here
+    r.__class__.cleanup()
+
+
+def test_serializers(r):
+    orig = r.parse_single("Ég fór niðrá bryggjuna.")
+    assert orig.tree is not None
+    assert orig.tree.match("S0 >> { IP > { VP > { PP > { P > { fs } } } } } ")
```

Comment:
Could also do a straight compare of `orig.tree.flat_with_all_variants` with a constant string